package.json

```diff
@@ -28,65 +27,54 @@
     }
   ],
   "bin": {
-    "bitcore-node": "./bin/bitcore-node"
+    "bitcore-node": "./bin/bitcore-node",
+    "bitcoind": "./bin/bitcoind"
   },
   "scripts": {
-    "install": "./bin/install",
-    "build": "./bin/build",
-    "clean": "./bin/clean",
-    "package": "node bin/package.js",
-    "upload": "node bin/upload.js",
-    "start": "node bin/start.js",
-    "test": "NODE_ENV=test mocha -R spec --recursive",
-    "coverage": "NODE_ENV=test istanbul cover _mocha -- --recursive",
-    "libbitcoind": "node bin/start-libbitcoind.js"
+    "preinstall": "./scripts/download",
+    "verify": "./scripts/download --skip-bitcoin-download --verify-bitcoin-download",
+    "test": "mocha -R spec --recursive",
+    "regtest": "./scripts/regtest",
+    "jshint": "jshint --reporter=node_modules/jshint-stylish ./lib",
+    "coverage": "istanbul cover _mocha -- --recursive"
   },
   "tags": [
     "bitcoin",
     "bitcoind"
   ],
   "dependencies": {
     "async": "^1.3.0",
-    "bindings": "^1.2.1",
+    "bitcoind-rpc": "braydonf/bitcoind-rpc#4850733b9806bc5e8e1508fa90f3c45782e6ee80",
```

Comment:
I "think" the sha should be https://github.com/bitpay/bitcoind-rpc/commit/381490bae5e3a8cefc0528b40666cd6980183499